apps/charterafrica/src/pages/api/v1/opportunities/consultation/[...service].page.js

```diff
@@ -0,0 +1,49 @@
+const documents = async (req, res) => {
+  const baseURL = "https://dc.sourceafrica.net/api/oembed.json";
+  const { url } = req.query;
+  try {
+    const fullURL = `${baseURL}?url=${url}`;
+    const response = await fetch(fullURL);
+    const data = await response.json();
+    res.status(200).json(data);
+  } catch (error) {
+    res.status(500).json({ error });
+  }
+};
+
+const youtube = async (req, res) => {
+  try {
+    const { service, ...rest } = req.query;
+    const url = service?.slice(1).join("/");
+    const key = process.env.GOOGLE_MAPS_API_KEY;
```

Comment:
```suggestion
    const key = process.env.GOOGLE_API_KEY;
```

Aren't we going to use a single Google API key for all supported services?

Reply:
@kilemensi please confirm that this change requires [updating this line turbo.json](https://github.com/CodeForAfrica/ui/blob/e138473c8f5f317d8453383878b7c9b9a20b3e9a/turbo.json#L89).

One key can be used provided it is properly enabled on console.

Reply:
Yes @koechkevin